README.md

```diff
@@ -18,6 +18,7 @@ pip install squape
 The package consists of several modules:
 - report - adds features to enhance the reporting and verification capabilities of Squish.
 - settings - allows using various test settings as context managers
+- video - allows capturing videos using a context manager with option to automatically delete a video when execution was successful
```

Comment:
The description should be more generic as more functionalities may come later. It should provide information about the idea not the implementation